contracts/dnssec-oracle/algorithms/EllipticCurve.sol

```diff
@@ -61,7 +61,7 @@ contract EllipticCurve {
     /**
      * @dev Transform affine coordinates into projective coordinates.
      */
-    function toProjectivePoint(uint x0, uint y0) public pure
+    function toProjectivePoint(uint x0, uint y0) internal pure
```

Comment:
Any reason you have to change to internal? I see no harm leaving it public so that people can inspect from Etherscan, etc

Reply:
As internal, the functions get compiled into the contract that uses the library, instead of having to call to an external library using a DELEGATECALL, so it saves gas.